html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div>
+        <label for="email">Email <br /> </label>
+        <input type="text" name="Email" id="email" />
+      </div>
+      <div>
+        <label for="Why is coding cool?">Why is coding cool? <br /></label>
+        <select name="Why is coding cool?" id="Why is coding cool?">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">
+            Because it's interesting
+          </option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <div>
+        <label for="Favorite Language"
+          >Favorite Language (Choose one) <br
+        /></label>
+        <input type="radio" name="Language" id="HTML" />
+        <label for="HTML">HTML</label>
+        <input type="radio" name="Language" id="CSS" />
+        <label for="CSS">CSS</label>
+        <input type="radio" name="Language" id="JavaScript" />
+        <label for="JavaScript">JavaScript</label>
+      </div>
+      <div>
+        <label for="Would you like to sign up for more information"
+          >Would you like to sign up for more information? <br />
+        </label>
+        <input type="checkbox" name="service" id="Sign me up!" />
+        <label for="Sign me up!">Sign me up!</label>
+      </div>
+      <div>
+        <label for="Comments">Comments <br /></label>
+        <textarea name="Comments" id="Comments" cols="30" rows="10"></textarea>
+      </div>
+      <div>
+        <input type="submit" id="submit" />
+        <label for="submit"></label>
```

Comment:
instead of a label here, you can actually put a `value` attribute on the input and set it to `Submit`